src/io/flutter/module/FlutterModuleBuilder.java

```diff
@@ -61,6 +65,15 @@ public void setupRootModel(ModifiableRootModel model) throws ConfigurationExcept
   }
 
   @Override
+  public ModuleWizardStep modifyStep(SettingsStep step) {
+    if (getModuleType() != null && step instanceof ProjectSettingsStep) {
```

Comment:
From the travis failure, it looks like ProjectSettingsStep doesn't exist in non-idea products. Is this whole class (FlutterModuleBuilder.java) idea specific? Perhaps we should call out such using idea-contribs.xml?